core/gesture.js

```diff
@@ -899,9 +899,7 @@ Blockly.Gesture.prototype.isBlockClick_ = function() {
  * @private
  */
 Blockly.Gesture.prototype.isFieldClick_ = function() {
-  var fieldEditable = this.startField_ ?
-      this.startField_.isCurrentlyEditable() : false;
-  return fieldEditable && !this.hasExceededDragRadius_ && (!this.flyout_ ||
+  return this.startField_ && !this.hasExceededDragRadius_ && (!this.flyout_ ||
```

Comment:
I'm suspicious of this change.  What happens now if you click on a non-editable field (such as a label)?  It should be a block click, not a field click, but I think that's no longer the case with this change.

Reply:
You are correct.

Options:
1) Revert change. If the image field's onClick parameter is defined set this.EDITABLE to true.
2) Revert change.  Have image field override bindEvents_ instead.

Reply:
Option 3 sounds good to me.

Reply:
Done!